proto/bpfd.proto

```diff
@@ -25,9 +25,9 @@ message LoadRequestCommon {
     } ;
     string section_name = 3;
     uint32 program_type = 4;
-    optional string id = 5;
+    optional string uuid = 5;
```

Comment:
++ I'm alright with that!

Reply:
The biggest negative would be that it's another peice of info that client's need to understand that's not conveyed by the API... i.e now they have to know the "bpfd.dev/uuid" is special.

Reply:
Define client in that statement 😏 
If you mean `bpfctl` - it doesn't need to understand the semantics of any of the K/V pairs. Just read/write them.

If you mean bpfd-agent/bpfd-operator, it can define it's own semantics on whatever it writes.
It could use the key "kittens" to store UUIDs if it wanted, although I wouldn't recommend it 😆

Reply:
Done!